### ISSUE TITLE
PG 10 issues

### ISSUE THREAD

**Issue by erthalion** [zalando/pg_view#83]
Hello,

I tried to use `pg_view` on PG10, and found two issues:

* at least on my installation `PG_VERSION` contains just `10`, so this code is causing an error about uninitialized variable `version`:

```python
if val is not None and len(val) >= 2:
    version = float(val)
else:
    dbname = get_dbname_from_path(pg_dir)
    postmasters[pg_dir] = [pid, version, dbname]
```

* `pg_xlog` was renamed to `pg_wal` (see the corresponding [message](https://www.postgresql.org/message-id/flat/CAB7nPqTu0y8uxbqXYjUObW9Oq4jnZxcPv_uTwstparBBjsrbQg%40mail.gmail.com#CAB7nPqTu0y8uxbqXYjUObW9Oq4jnZxcPv_uTwstparBBjsrbQg@mail.gmail.com) and [commit](https://github.com/postgres/postgres/commit/f82ec32ac30ae7e3ec7c84067192535b2ff8ec0e))

Of course it's easy to fix them, but I'm not sure how to do it in a backward compatible way.

**Comment by LappleApple** [comment]
Hello @erthalion, would you be up for making a bug fix/PR here? Then we can review and get this fixed faster for you.

**Comment by erthalion** [comment]
From what I see #84 should be enough for the "pg_wal" part, and we can forget about an issue with `PG_VERSION`, since on the current PG10 beta1 everything is fine.

**Comment by alexeyklyukin** [comment]
Fixed in #84
